util/src/filesystem.cpp

```diff
@@ -15,8 +18,34 @@ std::string binaryPathToInstallDir(const char* path)
   // of any symbolic links and relative references.
   // The path is then parented twice (from binary to "bin" folder, and then
   // the root).
-  return fs::canonical(fs::system_complete(fs::path(path)))
-    .parent_path().parent_path().string();
+  // If the path does not exist, the given binary is searched for in the PATH
+  // environment variable.
+
+  if (fs::exists(fs::system_complete(fs::path(path))))
+  {
+    return fs::canonical(fs::system_complete(fs::path(path)))
+      .parent_path().parent_path().string();
+  }
+  else
+  {
+    std::string webserverPath;
```

Comment:
This variable name is a little bit misleading. We can use this function to search not only webserver binary. So maybe a different name would be better, for example `pPath`.